docs/moderne-cli/README.md

```diff
@@ -64,10 +149,7 @@ With the CLI downloaded and configured, you're now ready to set up the repositor
    git checkout b527de52f5fd19f9fe550372c017d145a3b2a809
    ```
 
-3. If you tried building this repository right now, you would more than likely run into errors. This is because this
-   version of the Spring PetClinic repo requires Java 8. To ensure that everything builds correctly, you may need to
-   download Java 8 and update your `JAVA_HOME` environment variable. If you are on a Unix-based system, we recommend
-   using [SDKMan](https://sdkman.io/):
+3. If you tried building this repository right now, you would more than likely run into errors. This is because this version of the Spring PetClinic repo requires Java 8. To ensure that everything builds correctly, you may need to download Java 8 and update your `JAVA_HOME` environment variable. If you are on a Unix-based system, we recommend using [SDKMan](https://sdkman.io/):
```

Comment:
I'd used the IntelliJ auto format which splits lines by default; are you using a different auto formatter that's causing these changes?

Reply:
Ah ya I use VS Code instead. Maybe there's some type of format we can run on the code or some editor config file we can make to standardize?

Reply:
As discussed: disabled hard line wraps in IntelliJ as well, recommend others to do the same as that makes rewriting easier.